backend/src/impl/benchmark_configs/globalbench_ner.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "globalbench_ner",
+    "name": "GlobalBench NER",
+    "description": "A benchmark for measuring global progress in named entity recognition.",
+    "parent": "globalbench",
+    "system_query": {
+        "task": "named-entity-recognition"
```

Comment:
I think you want to change line 7 to "system_info.task_name": "named-entity-recognition".

Reply:
That's totally right! I thought about this a bit and in the end I decided to keep the json file the same, but add some code in the Python backend to call `findSystems` directly. This is mainly because I think asking the users creating a benchmark in JSON to use a verbose variable name like `system_info.task_name` might be a bit overkill.